.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+  - "3.6"      # current default Python on Travis CI
+  - "3.7"
+  - "3.8"
+  - "3.8-dev"  # 3.8 development branch
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
+# command to run tests
+script:
+  - pytest
```

Comment:
We'll configure our CI/CD pipeline with github action. Please remove this file.

Reply:
I already changed the project to use github actions.